python/python/glide/async_commands/core.py

```diff
@@ -1005,7 +1005,7 @@ async def expire(
 
         Examples:
             >>> await client.expire("my_key", 60)
-                1  # Indicates that a timeout of 60 seconds has been set for "my_key."
+                True  # Indicates that a timeout of 60 seconds has been set for "my_key."
```

Comment:
May be move doc fixes to another PR?

Reply:
+1 for next time, for now just make sure to have these changes in a separate commit